docs/understanding-your-pipeline/canonical-event/index.md

```diff
@@ -36,274 +42,490 @@ The Snowplow data table is designed to be immutable: the data in each line shoul
+| `br_colordepth`                                           | int       | Bit depth of the browser color palette                                     | No    | 24                                                             | Tracking                                                                                                                                                                                          |
+| `br_viewheight`                                           | int       | Viewport height                                                            | No    | 1000                                                           | Tracking                                                                                                                                                                                          |
+| `br_viewwidth`                                            | int       | Viewport width                                                             | No    | 1000                                                           | Tracking                                                                                                                                                                                          |
 
 ### Event-specific fields
 
 Snowplow includes specific fields to capture data associated with specific events.
 
 Note that to date, all event types have been defined by Snowplow. Also note that `event_vendor` values follow the [Java package naming convention](http://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html).
 
-Snowplow currently supports (or will support in the near future) the following event types:
+Snowplow currently supports the following event types:
 
-| **Event type** | **Value of `event` field in model** |
-| --- | --- |
-| [Page views](#page-views) | 'page_view' |
-| [Page pings](#page-pings) | 'page_ping' |
-| [E-commerce transactions](#e-commerce-transactions) | 'transaction' and 'transaction_item' |
-| [Custom structured events](#custom-structured-events) | 'struct' |
-|[Custom unstructured events](#custom-unstructured-events) | 'unstruct' |
+| Event type                                                | Value of `event` field in model      |
+| --------------------------------------------------------- | ------------------------------------ |
+| [Page views](#page-views)                                 | 'page_view'                          |
+| [Page pings](#page-pings)                                 | 'page_ping'                          |
+| [E-commerce transactions](#e-commerce-transactions)       | 'transaction' and 'transaction_item' |
+| [Custom structured events](#custom-structured-events)     | 'struct'                             |
+| [Custom unstructured events](#custom-unstructured-events) | 'unstruct'                           |
```

Comment:
Link is broken with the header change from `#### Custom unstructured events` > `#### Custom Self-Describing (unstructured) events` below , might need to specify an ID on that heading or update this link to `#custom-self-describing-unstructured-events`. I wish docusaurus was smart enough to catch this. :/

Reply:
Yeah I spent like half a day the other week trying to find a tool to identify broken links with/of headers, but couldn't find something that works with our link structure...